### PR TITLE
Fix id and all endpoints

### PR DESCRIPTION
**Pull request by carlosroman**

### CODE REVIEW

File: people/handlers.go [hunk 1]
```diff
@@ -20,15 +21,55 @@ func NewPeopleHandler(service PeopleService) PeopleHandler {
...
+
+	pv, err := h.service.getPeopleByUUID()
+
+	if err != nil {
+		writeJSONMessageWithStatus(writer, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	defer pv.Close()
+	writer.WriteHeader(http.StatusOK)
+	log.Info("Status written")
```
Comment (ghost) [inline people/handlers.go]: Is this log message needed?

File: people/handlers.go [hunk 2]
```diff
@@ -20,15 +21,55 @@ func NewPeopleHandler(service PeopleService) PeopleHandler {
...
+
+	pv, err := h.service.getPeople()
+
+	if err != nil {
+		writeJSONMessageWithStatus(writer, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	defer pv.Close()
+	writer.WriteHeader(http.StatusOK)
+	log.Info("Status written")
```
Comment (ghost) [inline people/handlers.go]: Is this log message needed?

File: people/handlers.go [hunk 3]
```diff
@@ -20,15 +21,55 @@ func NewPeopleHandler(service PeopleService) PeopleHandler {
...
+	io.Copy(writer, &pv)
+}
+
+func (h *PeopleHandler) GetPeopleByUUID(writer http.ResponseWriter, req *http.Request) {
```
Comment (ghost) [inline people/handlers.go]: Calling this GetPeopleByUUID makes me think I'd need to pass in a UUID, which isn't the case. Perhaps GetPeopleUUIDs?

File: people/handlers_test.go [hunk 4]
```diff
@@ -263,14 +302,56 @@ type dummyService struct {
 	wg           *sync.WaitGroup
 }
 
-func (s *dummyService) getPeople() ([]personLink, bool) {
+func (s *dummyService) getPeopleOld() ([]personLink, bool) {
```
Comment (ghost) [inline people/handlers_test.go]: If this isn't used, we can just remove it?

File: people/handlers_test.go [hunk 5]
```diff
@@ -263,14 +302,56 @@ type dummyService struct {
...
+
+func (s *dummyService) getPeopleLink() (io.PipeReader, error) {
+	pv, pw := io.Pipe()
+	go func() {
+		var links []personLink
+		for _, sub := range s.people {
+			links = append(links, personLink{APIURL: "http://localhost:8080/transformers/people/" + sub.UUID})
+		}
+		b, _ := json.Marshal(links)
+		log.Infof("Writting bytes... %v", string(b))
```
Comment (ghost) [inline people/handlers_test.go]: Spelling

File: people/handlers_test.go [hunk 6]
```diff
@@ -263,14 +302,56 @@ type dummyService struct {
...
+
+func (s *dummyService) getPeopleByUUID() (io.PipeReader, error) {
+	pv, pw := io.Pipe()
+	go func() {
+		var links []personUUID
+		for _, sub := range s.people {
+			links = append(links, personUUID{UUID: sub.UUID})
+		}
+		b, _ := json.Marshal(links)
+		log.Infof("Writting bytes... %v", string(b))
```
Comment (ghost) [inline people/handlers_test.go]: Spelling

File: people/handlers.go (hunk 2 above)
Comment (carlosroman) [reply people/handlers.go]: Nope... was for debug... removing...

File: people/handlers.go (hunk 3 above)
Comment (carlosroman) [reply people/handlers.go]: And that is why I'm not allowed to name things :)

File: people/handlers_test.go (hunk 4 above)
Comment (carlosroman) [reply people/handlers_test.go]: Was going to keep it as got a bad feeling that style of "links" might be used by another service. Will rename it to something more sensible.

Comment (carlosroman) [reply people/handlers_test.go]: oh, this was a test... removing.

**Review by kovacshuni — APPROVED** [review]

**Pull request merged**